### ISSUE TITLE
showoff create or showoff create init?

### ISSUE THREAD

**Issue by marksands** [schacon/showoff#23]
In version 1.0 I've always done `showoff create dir` but then I upgrade to 1.4 and now I have to do `showoff create init dir` instead. 

Rather, showoff insists that I do it the old way:

```
usage: showoff create (directory)

this command helps start a new showoff presentation by setting up the
proper directory structure for you.  it takes the directory name you would
like showoff to create for you.
```

Note the absence of `init`.

**Comment by marksands** [comment]
On further inspection it appears all the showoff commands need to be followed by `init`.

**Comment by davetron5000** [comment]
This is not happening for me on trunk

```
> showoff create foo
done. run 'showoff serve' in foo/ dir to see slideshow
> rm -rf foo
> showoff init foo
done. run 'showoff serve' in foo/ dir to see slideshow
```

It appears that v0.1.4 _does_ do this.  So, I think the answer is that this will be fixed in v0.1.5...

**Comment by marksands** [comment]
Huh. Yeah, looks like this is fixed in the trunk. I guess I'll stick with this build until a new gem version is published. Thanks for the tip!
